### ISSUE TITLE
NodeHasPkgCount.java:34-39: This matcher should also...

### ISSUE THREAD

**Issue by 0pdd** [artipie/rpm-adapter#307]
The puzzle `285-b9db6ed3` from #285 has to be resolved: 

https://github.com/artipie/rpm-adapter/blob/ad868fd583a12f7cc8fe389cad0951ff17e6ad5d/src/test/java/com/artipie/rpm/hm/NodeHasPkgCount.java#L34-L39

The puzzle was created by paulodamaso on 25-Jun-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8), @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#307](https://github.com/artipie/rpm-adapter/issues/307) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/ca84df95-8255-41f7-8b89-153cd1b6fa0e); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@8366e303-9dc5-4679-a29a-c61b61f06cfb, thread: PQ-CT2E6TK9B
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 2 error(s) in XML document: -1:-1: cvc-pattern-valid: Value 'gh:Week 27' is not facet-valid with respect to pattern '[a-z]{2}:[A-Z0-9a-z.\-#/]+' for type 'milestoneID'.;-1:-1: cvc-complex-type.2.2: Element 'successor' must have no element [children], and the value must be valid.
```

[0.53.15](https://github.com/zerocracy/farm): CID: [bfe8ff0c-4b85-4ac5-928a-ddc10b20b5ca](https://www.0crat.com/footprint/CT2E6TK9B/bfe8ff0c-4b85-4ac5-928a-ddc10b20b5ca), Type: "Job milestoned"

**Comment by 0pdd** [comment]
The puzzle `285-b9db6ed3` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#311](https://github.com/artipie/rpm-adapter/issues/311) are still not solved.

**Comment by 0crat** [comment]
Job was finished in 44 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317) are still not solved; solved: [#311](https://github.com/artipie/rpm-adapter/issues/311).

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#359](https://github.com/artipie/rpm-adapter/issues/359), [#360](https://github.com/artipie/rpm-adapter/issues/360) are still not solved; solved: [#311](https://github.com/artipie/rpm-adapter/issues/311), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#310](https://github.com/artipie/rpm-adapter/issues/310), [#360](https://github.com/artipie/rpm-adapter/issues/360) are still not solved; solved: [#311](https://github.com/artipie/rpm-adapter/issues/311), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317), [#359](https://github.com/artipie/rpm-adapter/issues/359).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#360](https://github.com/artipie/rpm-adapter/issues/360) is still not solved; solved: [#310](https://github.com/artipie/rpm-adapter/issues/310), [#311](https://github.com/artipie/rpm-adapter/issues/311), [#316](https://github.com/artipie/rpm-adapter/issues/316), [#317](https://github.com/artipie/rpm-adapter/issues/317), [#359](https://github.com/artipie/rpm-adapter/issues/359).
